models/contacts.js

```diff
@@ -1,19 +1,75 @@
-// const fs = require('fs/promises')
+const { nanoid } = require('nanoid');
 
-const listContacts = async () => {}
+const fs = require('fs/promises');
+const path = require('path');
 
-const getContactById = async (contactId) => {}
+const contactsPath = path.join(__dirname, 'contacts.json');
 
-const removeContact = async (contactId) => {}
+/**
+ * Reads and returns an array of contacts from a file
+ * @returns array
+ */
+async function listContacts() {
+  const contacts = await fs.readFile(contactsPath);
+  return JSON.parse(contacts);
+}
+
+/**
+ * Finds a contact object by id and returns it
+ * @param {string} contactId
+ * @returns object
+ */
+async function getContactById(contactId) {
```

Comment:
```suggestion
async function getContactByIdService(contactId) {
``` аналогічно для інших